.github/workflows/pr_title.yml

```
@@ -0,0 +1,42 @@
+name: 'PR is Conventional and Semantic'
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+    branches:
+      - main
+        
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  conventional_pr_title:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5.5.3
+        with: 
+          scopes: |
+            llc
+            ui
+            doc
+            repo
+            samples
```

let's only keep llc and repo for now.
```suggestion
            llc
            repo
```